package.json

```diff
@@ -278,6 +278,11 @@
                                 "description": "This setting can be used to override the gdb-server path user/workspace setting for a particular launch configuration. It is the full pathname to the executable or name of executable if it is in your PATH",
                                 "type": "string"
                             },
+                            "serverArgs": {
+                                "default": null,
```

Comment:
Just like the items that are an array of string items, you might want to
```JSON
                                "default": [],
                                "type": "array",
                                "items": "string",
```
to get better JSON validation in the editor.